source/Coop/Mod/CoopClient.cs

```diff
@@ -113,15 +118,13 @@ private void ConnectionCreated(ConnectionClient con)
 
             m_ReconnectAttempts = MaxReconnectAttempts;
             TryInitPersistence(con);
-            SyncedObjectStore = new RemoteStore(m_SyncedObjects, con);
```

Comment:
Is there an issue with the store? As it's currently not in use, i don't mind removing it. Just wondering.

Reply:
No. It's just my fail.